cards/src/main/resources/cards/custom/group3/set_1087/minion_hydraquatom.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Hydraquatom",
+  "baseManaCost": 8,
+  "type": "MINION",
+  "heroClass": "RUST",
+  "race": "MECH",
+  "baseAttack": 8,
+  "baseHp": 8,
+  "rarity": "LEGENDARY",
+  "description": "Deathrattle: Fire four missiles at random enemies that deals 4 damage each.",
```

Comment:
`Aftermath: Fire four missiles at random enemies that deal 4 damage each.`

^deathrattle to aftermath, deals to deal